tools/test_utils.py

```diff
@@ -311,32 +321,38 @@ def run_resnet50(build_dir):
     os.chdir(root_pwd)
 
 
-def run_resnet50_from_artifacts(artifact_dir, batch_size, iterations):
+def run_resnet50_from_artifacts(ngraph_tf_src_dir, artifact_dir, batch_size,
+                                iterations):
 
     root_pwd = os.getcwd()
     artifact_dir = os.path.abspath(artifact_dir)
-
+    ngraph_tf_src_dir = os.path.abspath(ngraph_tf_src_dir)
     install_ngraph_bridge(artifact_dir)
 
+    patch_file = os.path.abspath(
+        os.path.join(ngraph_tf_src_dir, "test/grappler/benchmark_cnn.patch"))
+
     # Now clone the repo and proceed
     call(['git', 'clone', 'https://github.com/tensorflow/benchmarks.git'])
-    os.chdir('benchmarks/scripts/tf_cnn_benchmarks/')
-
+    os.chdir('benchmarks')
     call(['git', 'checkout', '4c7b09ad87bbfc4b1f89650bcee40b3fc5e7dfed'])
 
+    # Check to see if we need to patch the repo for Grappler
+    import tensorflow as tf
+    import ngraph_bridge
+    if ngraph_bridge.is_grappler_enabled():
+        print("Patching repo using: %s" % patch_file)
+        result = call(["patch", "-p1", "-N", "-i", patch_file])
+        print("Patch result: %d" % result)
```

Comment:
please use `command_executor` as it catches and raises exceptions if the command did not finish successfully.